react-native/react/pinentry/index.js

```diff
@@ -1,7 +1,7 @@
 'use strict'
 
 import React, { Component } from '../base-react'
-import PinentryRender from './pinentry-render'
+import PinentryRender from './pinentry.render'
```

Comment:
had to fixup pinentry so mobile builds, also going to make all the names blah.render.js everywhere soon